example/src/screens/Meeting.tsx

```diff
@@ -266,18 +276,32 @@ const Meeting = ({
             clearMessageRequest();
             navigate('WelcomeScreen');
           }}>
-          <Feather name="phone-off" style={styles.leaveIcon} size={30} />
+          <Feather
+            name="phone-off"
+            style={styles.leaveIcon}
+            size={dimension.viewHeight(30)}
+          />
         </TouchableOpacity>
         <TouchableOpacity
           style={styles.singleIconContainer}
           onPress={() => {
             setModalVisible(true);
           }}>
-          <Feather name="message-circle" style={styles.videoIcon} size={30} />
-          {/* <Image
-            source={require('../assets/flipCamera.svg')}
-            style={styles.cameraImage}
-          /> */}
+          <Feather
+            name="message-circle"
+            style={styles.videoIcon}
+            size={dimension.viewHeight(30)}
+          />
+        </TouchableOpacity>
+        <TouchableOpacity
+          style={styles.singleIconContainer}
+          onPress={() => {
+            instance.localPeer.localVideoTrack().switchCamera();
+          }}>
+          <Image
```

Comment:
can't we get this icon from vector icons?

Reply:
did not find any icon there